packages/metal-throttle/.jshintrc

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
We don't need this file at all, we're eslinting it :)

Reply:
Done